README.md

```diff
@@ -77,6 +77,8 @@
 - **[Create_post]**
 - **[Create_comment]**
 - **[Like_post]**
+- **[Optimize_query]**
```

Comment:
- [ ] In your post `index` action in the `posts_controller.rb` you need to fix N+1 query problems when fetching the comments and post. Currently, in your posts index action, you are not fetch the comments as well. Kindly fetch the comments along with the posts. See the example below - you can then tweak it.

  ```
  @posts = @user.posts.includes(:comments)
  ```

  ![image](https://github.com/Ezzywealth/rails_blog/assets/36542986/e4b4fd53-f5fd-40ce-8ab0-d37f61f5f16f)